[2022-04-07 22:23:42,221] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_badges.create_badges_stack_overflow_data_external_table scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2022-04-07 22:23:42,375] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: load_to_BigQuery.create_external_badges.create_badges_stack_overflow_data_external_table scheduled__2022-03-01T00:00:00+00:00 [queued]>
[2022-04-07 22:23:42,375] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 22:23:42,375] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-07 22:23:42,375] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-07 22:23:43,239] {taskinstance.py:1270} INFO - Executing <Task(BigQueryCreateExternalTableOperator): create_external_badges.create_badges_stack_overflow_data_external_table> on 2022-03-01 00:00:00+00:00
[2022-04-07 22:23:43,329] {standard_task_runner.py:52} INFO - Started process 9204 to run task
[2022-04-07 22:23:43,378] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'load_to_BigQuery', 'create_external_badges.create_badges_stack_overflow_data_external_table', 'scheduled__2022-03-01T00:00:00+00:00', '--job-id', '14614', '--raw', '--subdir', 'DAGS_FOLDER/load_to_BigQuery_dag.py', '--cfg-path', '/tmp/tmp5sd7az3p', '--error-file', '/tmp/tmp6x7s_nj7']
[2022-04-07 22:23:43,379] {standard_task_runner.py:80} INFO - Job 14614: Subtask create_external_badges.create_badges_stack_overflow_data_external_table
[2022-04-07 22:23:43,808] {logging_mixin.py:109} INFO - Running <TaskInstance: load_to_BigQuery.create_external_badges.create_badges_stack_overflow_data_external_table scheduled__2022-03-01T00:00:00+00:00 [running]> on host 36107d30f64e
[2022-04-07 22:23:44,064] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-07 22:23:44,124] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_to_BigQuery
AIRFLOW_CTX_TASK_ID=create_external_badges.create_badges_stack_overflow_data_external_table
AIRFLOW_CTX_EXECUTION_DATE=2022-03-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-01T00:00:00+00:00
[2022-04-07 22:23:44,126] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-04-07 22:23:44,671] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=load_to_BigQuery, task_id=create_external_badges.create_badges_stack_overflow_data_external_table, execution_date=20220301T000000, start_date=20220407T222342, end_date=20220407T222344
[2022-04-07 22:23:44,740] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-07 22:23:44,858] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
